contributing_to_docs/doc_guidelines.adoc

```diff
@@ -254,11 +261,11 @@ IMPORTANT: You cannot link to a repository that is hosted on www.github.com.
 
 TIP: If you want to build a link from a URL _without_ changing the text from the actual URL, just print the URL without adding a `[friendly text]` block at the end; it will automatically be rendered as a link.
 
-=== Links to Internal Topics
+=== Links to internal content
 There probably are three scenarios for linking to other content:
 
-1. Link to another topic file that exists in the same topic group, or directory.
-2. Link to another topic file that exists in a separate topic group, or directory.
+1. Link to another file that exists in the same group, or directory.
```

Comment:
For this whole section, I'd make it explicit that links are just to assemblies and simplify the references to directories. (I'd rather have to make the language generic if we can xref in modules later than have people skip the warning, try to xref in a module, then argue about it.)

s/file/assembly
s/group, or directory/directory